packages/react/README.md

```diff
@@ -0,0 +1,136 @@
+# Metrom React
+
+Collection of React hooks a for seamless and efficient integration with Metrom.
+
+## Installation
+
+The hooks rely on [wagmi](https://wagmi.sh/react/installation#installation)
+under the hood, so a few required packages must be installed, alongside
+`@metrom-xyz/react`:
+
+```bash
+pnpm add @metrom-xyz/react wagmi viem @tanstack/react-query
+# or
+yarn add @metrom-xyz/react wagmi viem @tanstack/react-query
+# or
+npm add @metrom-xyz/react wagmi viem @tanstack/react-query
+```
+
+## Setup
+
+A couple configuration steps are required before being able to use the Metrom
+hooks.
+
+### External dependencies
+
+Once installed, set up the Wagmi and TanStack configuration, then wrap your
+React app with the required context providers by following the official
+[wagmi documentation](https://wagmi.sh/react/getting-started#manual-installation).
+
+### Wrap app in context provider
+
+Wrap your app with the `MetromProvider` React Context Provider and pass the
+environment you wish to target the `environment` property.
+
+The `Environment` is exported by the library.
+
+```tsx
+import { Environment, MetromProvider } from "@metrom-xyz/react";
+
+function App() {
+  return (
+    <MetromProvider environment={Environment.Development}>
+      {/** ... */}
+    </MetromProvider>
+  );
+}
+```
+
+> Omitting the environment will default to `production`.
```

Comment:
Should we actually default to dev in this case? Feels like the better option

Reply:
I was viewing from the 3rd party perspective and thought it would be better to default on production, but we can do the opposite, quick change.

Reply:
Addressed in 4abc4d15fd87090553ecbcbcc5b48b1224ae4b24.